website/content/en/docs/main/tutorials/shared/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: shared
+description: Shared content between the tutorials.
+weight: 40
+draft: true
```

Comment:
Draft is fine during development but should be removed from the frontmatter eventually...

Reply:
Not sure this page bundle file is really needed.
What was the rationale for inclusion?

Reply:
can these `shared` files be sourced from `website/static/files/tutorials` instead?
I think all static resources should go there.